src/Rules/Drupal/RenderCallbackRule.php

```diff
@@ -217,7 +199,16 @@ private function doProcessNode(Node\Expr $node, Scope $scope, string $keyChecked
     private function getType(Node\Expr $node, Scope $scope):  Type
     {
         $type = $scope->getType($node);
-        if ($type instanceof ConstantStringType) {
+        if ($type instanceof IntersectionType) {
+            // Covers concatenation of static::class . '::methodName'.
+            if ($node instanceof Node\Expr\BinaryOp\Concat) {
+                $leftType = $scope->getType($node->left);
+                $rightType = $scope->getType($node->right);
+                if ($leftType instanceof GenericClassStringType && $leftType->getGenericType() instanceof StaticType && $rightType instanceof ConstantStringType) {
+                    return new ConstantStringType($leftType->getGenericType()->getClassName() . $rightType->getValue());
+                }
```

Comment:
One problem we'll have is this in \PHPStan\Type\Constant\ConstantStringType::isCallable

```
        // 'MyClass::myStaticFunction'
        $matches = \_PHPStan_4b01b3801\Nette\Utils\Strings::match($this->value, '#^([a-zA-Z_\\x7f-\\xff\\\\][a-zA-Z0-9_\\x7f-\\xff\\\\]*)::([a-zA-Z_\\x7f-\\xff][a-zA-Z0-9_\\x7f-\\xff]*)\\z#');
        if ($matches !== null) {
            if (!$reflectionProvider->hasClass($matches[1])) {
                return \PHPStan\TrinaryLogic::createMaybe();
            }
```

Since we have a `static` keyword, the result will always be "maybe". That's how PHPStan works. The reflection provider will return `false` on `hasClass`.